app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -161,16 +161,25 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
+        //link to string resources
+        String title_resource = context.getResources().getString(R.string.nominating_for_Deletion);
+        String success_resource = context.getResources().getString(R.string.success);
+        String successfully_nominated_resource = context.getResources().getString(R.string.successfully_nominated);
+        String for_deletion_resource = context.getResources().getString(R.string.for_deletion);
+        String failed_resource = context.getResources().getString(R.string.failed);
+        String could_not_request_deletion_resource = context.getResources().getString(R.string.could_not_request_deletion);
+
         String message;
-        String title = "Nominating for Deletion";
+        String title = title_resource;
 
         if (result){
-            title += ": Success";
-            message = "Successfully nominated " + media.getDisplayTitle() + " deletion.";
+            title += ":"+success_resource;
+            message = successfully_nominated_resource + media.getDisplayTitle() + for_deletion_resource+".";
         }
         else {
-            title += ": Failed";
-            message = "Could not request deletion.";
+            title += ":"+failed_resource;
+            message = could_not_request_deletion_resource+".";
```

Comment:
In Japanese the point is a different character, so it should be localized too :-)